examples/LaserWakefield/include/simulation_defines/param/densityConfig.param

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2013-2017 Axel Huebl, Rene Widera, Felix Schmitt,
+ *                     Richard Pausch, Marco Garten
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "particles/densityProfiles/profiles.def"
+/* preprocessor struct generator */
+#include "preprocessor/struct.hpp"
+
+
+namespace picongpu
+{
+namespace SI
+{
+    /** Base density in particles per m^3 in the density profiles.
+     *
+     * This is often taken as reference maximum density in normalized profiles.
+     * Individual particle species can define a `densityRatio` flag relative
+     * to this value.
+     *
+     * unit: ELEMENTS/m^3
+     */
+    constexpr float_64 BASE_DENSITY_SI = 1.e25;
+}
+
+namespace densityProfiles
+{
+    PMACC_STRUCT(GaussianParameter,
+        /** Profile Formula:
+         *   constexpr float_X exponent = abs((y - gasCenter_SI) / gasSigma_SI);
```

Comment:
OPTIONAL:
Comment still contains `gas` (here it is OK, since it is really a gas) - however @ax3l do you want to adjust this for consistency? 

also lines: 49, 51, 52, 53, 55, 56, 67, 70, 71, 73, 76, 77